src/devices/Is31fl3730/MatrixMode.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Device.Gpio;
+using System.Device.Spi;
+using Iot.Device.Multiplexing.Utility;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Represents an IS31FL3731 LED Matrix driver
+    /// </summary>
+    public enum MatrixMode
+    {
+        /// <summary>
+        /// Represents a 8x8 LED matrix.
+        /// </summary>
+        Size8x8,
```

Comment:
As you are using explicit check of MatrixMode > 0 earlier, make the first element explicit to 0